content/sales.rst

```diff
@@ -0,0 +1,152 @@
+
+For those involved in procurement, please contact the Sales team.
+
+**Pre-demo session:**
+
+We recommend obtaining the trial version and installing until the
+`initial stage <https://www.sftpplus.com/documentation/sftpplus/latest/getting-started.html>`_
+
+In that way, you are able to continue after the demo session.
+
+
+Items to consider agreeing to a demo:
+-------------------------------------
+
+Is the customer actively pursuing a trial or considering a trial?
+
+If they are still considering a trial, it would be best to get a list of
+requirements first, and send through response.
+
+
+Is it attended by administrators / individuals involved in managing?
+--------------------------------------------------------------------
+
+Like readership of the documentation page, the session should be attended by
+at least an administrator or individual involved in managing the roll-out.
+
+This will increase the chance of knowledge transfer inside their organization
+during the trial and decrease instances of conducting more than one session.
+
+It will also smooth the process when the customer does the initial
+configuration during the trial.
+
+
+When demo session is agreed ask for the following:
+--------------------------------------------------
+
+* Dates available
+* Times available (and local timezone, so that we know what GoToMeeting
+  toll free / long distance numbers to provide)
+* Number of attendees (max 4 attendees incl 1 organiser and 1 co-organiser in
+  case or max 5 attendees if 1 organiser)
+* Platform
+* Protocols
+* List or criteria of requirements to know what to set up in advance
+
+
+Sending out demo session details
+--------------------------------
+
+* Session is held in the first 10 minutes. After that it may be rescheduled.
+
+* Phone bridge is recommended, if not, use a VOIP headset.
+
+
+Marketing / Promotional Materials
+=================================
+
+Promotions and marketing materials are mainly located in the main website.
+
+It should be as generic and non-technical as possible with links to the
+Documentation for more in-depth / technical information.
+
+To save materials, or to obtain past materials, the `chevah/design` repository
+is available.
+Please ask the team for access to the repository.
+
+
+Announcing a new release to the email list
+------------------------------------------
+
+After the website is updated and News item published, we send a newsletter:
+
+1. Go to Campaigns in Mailchimp.
+
+2. Select 'Replicate' besides 'NEW: ACME Release Announcement'.
+   If it is a security release, use the Security Advisories email list.
+
+3. Select the News Announcements email list.
+
+4. Update the subject and email with the News text used to announce the
+   new release. You can use the text in the News article.
+
+5. Select Send. Before sending the final email, preview first by going
+   to 'Preview and Send' on the top menu. Select 'Send a test email'.
+
+
+Mock ups and designs for the website
```

Comment:
Is this "sales related" ?

I think it should be removed from this PR and maybe move into documentation or somewhere else